Applications/Spire/Include/Spire/Ui/SideBox.hpp

```diff
@@ -0,0 +1,18 @@
+#ifndef SPIRE_SIDE_BOX_HPP
+#define SPIRE_SIDE_BOX_HPP
+#include "Nexus/Definitions/Side.hpp"
+#include "Spire/Ui/DropDownBoxAdaptor.hpp"
+#include "Spire/Ui/Ui.hpp"
+
+namespace Spire {
+
+  template<>
+  void DropDownBoxAdaptor<Nexus::Side>::populate_data() {
```

Comment:
This is effectively hard coding the model, have ```DropDownBoxAdaptor``` take a model in its constructor. It's not true that every ```DropDownBox``` for a given type always displays the same hardcoded values.